store/src/main/java/com/nytimes/android/external/store/base/impl/StoreInitializer.java

```diff
@@ -0,0 +1,52 @@
+package com.nytimes.android.external.store.base.impl;
+
+import com.nytimes.android.external.cache.Cache;
+import com.nytimes.android.external.cache.CacheBuilder;
+
+import java.util.concurrent.TimeUnit;
+
+import rx.Observable;
+
+final class StoreInitializer {
```

Comment:
Let's rename this to `CacheFactory` and have methods  `createCache`, `createInflighter`